### ISSUE TITLE
Headings are not structured - Admissions

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9391]
# Motivation

Siteimprove is reporting an accessibility best practice issue on Admissions area of study pages. One or more heading levels have been skipped.

The information block in the sidebar has an `<h4>` and the previous heading is an `<h1>`.

https://my2.siteimprove.com/Inspector/998093/A11Y/Page?pageId=44268464314&impmd=BA8559B81DC67FCE2867D1E363B05A58&issue=cantTell+failed&wcag=twopointtwo#/sia-r53/failed/SUNkQ0JPelI=;MWVhZWJjZGM=

**Comment by samyukthach** [comment]
Hi, I’m an Accessibility Specialist. I’d like to help with this issue and review it against WCAG guidelines.
